extensions/database/src/main/java/com/google/android/agera/database/SqlRequestCompiler.java

```diff
@@ -112,16 +121,40 @@ public Object arguments(@NonNull final String... arguments) {
 
```

Comment:
how to switch back to `CONFLICT_NONE `?
also `onConflict(int conflictAlgorithm)` could be useful